### ISSUE TITLE
Automation Recurring Time

### ISSUE THREAD

**Issue by belevine** [nightscout/AndroidAPS#3112]
AAPS 3.2.0.2

In automations, it is not possible to create a time range for recurring times.  This would be useful to select only specific days of the week to have an action taken during a time range.  Currently, recurring time automations can only be set to occur at a specific time, and time ranges occur on all days. 

This request is to have the day of week selection added to the time range trigger.

**Comment by TimVroom** [comment]
At the moment I have a locally edited version that includes this:
Just a simplified version of the recurring time trigger which you can only select the date.
This works great in combination with a time range or any of the other triggers for me.

So my question is:

why the need to include the other already existing triggers into one for recurring purposes? Isn't a single day selector more then enough?

I could make a PR for this, but im not sure about all the inner workings, so not sure it would be accepted. (will link this issue just in case)

**Comment by belevine** [comment]
> At the moment I have a locally edited version that includes this: Just a simplified version of the recurring time trigger which you can only select the date. This works great in combination with a time range or any of the other triggers for me.
> 
> So my question is:
> 
> why the need to include the other already existing triggers into one for recurring purposes? Isn't a single day selector more then enough?
> 
> I could make a PR for this if there is any interest, but im not sure about all the inner workings, so not sure it would be accepted. (will link this issue just in case)

Yes, a day selector in combination with a time range is what I am looking to do. I can't figure out how to do that with the current triggers; is it possible?

**Comment by TimVroom** [comment]
@belevine Seems like the PR is accepted: so in the upcomming version it should be working.
So with the current stable version it is not possible (as far as i know)

**Comment by belevine** [comment]
> @belevine Seems like the PR is accepted: so in the upcomming version it should be working. So with the current 3.2.x version it is not possible (as far as i know)

Thank you!
